rename.js

```diff
@@ -0,0 +1,17 @@
+const fs = require("fs");
+const path = require("path");
+
+fs.readdir("./esm", (err, files) => {
```

Comment:
Can you tell a bit more what is purpose of this. Not fully understanding why this rename is necessary - could it be avoided?